src/bin/main.rs

```diff
@@ -60,6 +60,10 @@ pub struct NewArgs {
     /// This can be a GitHub repository (`user/repo`) or a full Git URL.
     ///
     /// Can be omitted to use a built-in template.
```

Comment:
If omitted, it still uses a github template, which is not more or less builtin than `2d`. Could we rephrase this?